vars/longevityPipeline.groovy

```diff
@@ -51,23 +51,6 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('email_recipients', 'qa@scylladb.com')}",
                    description: 'email recipients of email report',
                    name: 'email_recipients')
-
-            string(defaultValue: "${pipelineParams.get('ami_id_monitor', '')}",
```

Comment:
Please also remove the references to those values in the step below

Reply:
done